src/nomad_simulations/schema_packages/numerical_settings.py

```diff
@@ -123,25 +109,114 @@ class Mesh(ArchiveSection):
         shape=['n_points'],
         description="""
         The amount of times the same point reappears. A value larger than 1, typically indicates
-        a symmetry operation that was applied to the `Mesh`. This quantity is equivalent to `weights`:
+        a symmetry operation that was applied to the `Mesh`.
```

Comment:
Does this imply then that `points` are symmetry-reduced? This unclarity caused several issues in our old schema.
Pls modify `points` to become `reduced_points` and `irreduced_points` (or some variant of this naming). The former does not have to be fully symmetry-reduced. It should be made clear that `multiplicities` and `weights` refer to `reduced_points`.